api/src/main/java/org/eclipse/microprofile/concurrent/spi/ConcurrencyProvider.java

```diff
@@ -63,10 +63,21 @@ public static ConcurrencyProvider instance() {
     ThreadContextBuilder newThreadContextBuilder(); 
 
     /**
-     * Allows the container to register the <code>ConcurrencyProvider</code>
+     * <p>Allows the container to register the <code>ConcurrencyProvider</code>
      * implementation. At most one implementation can be registered at any
      * given point in time. In order to register a different implementation,
-     * the container must first unregister its previous implementation.
+     * the container must first unregister its previous implementation.</p>
+     *
+     * <p>Upon successful registration of the <code>ConcurrencyProvider</code>,
+     * the MicroProfile Concurrency implementation must subsequently query the
+     * container's <code>ServiceLoader</code> for all
```

Comment:
Is it then legit to have more than one `ConcurrencyProvider` in a running application?

Reply:
No it's explicitly forbidden.

Reply:
Depends, reading on a `register()` method, it is forbidden there.
See [here](https://github.com/eclipse/microprofile-concurrency/blob/master/api/src/main/java/org/eclipse/microprofile/concurrent/spi/ConcurrencyProvider.java#L100).

And I think it doesn't make much sense to have more than one there.

Reply:
Fully agree it is explicitly forbidden to have more than one ConcurrencyProvider.  Just let me know if you see anywhere that suggests otherwise, and I'll be happy to get it updated.